app/models/concerns/inquiry/addon/input_setting.rb

```diff
@@ -9,15 +9,17 @@ module InputSetting
       field :required, type: String, default: "required"
       field :additional_attr, type: String, default: ""
       field :input_confirm, type: String, default: ""
-      permit_params :input_type, :required, :additional_attr, :select_options, :input_confirm
+      field :max_upload_file_size, type: Integer, default: 0
+      permit_params :input_type, :required, :additional_attr, :select_options, :input_confirm, :max_upload_file_size
 
-      validates :input_type, presence: true, inclusion: { in: %w(text_field text_area email_field radio_button select check_box) }
+      validates :input_type, presence: true, inclusion: { in: %w(text_field text_area email_field radio_button select check_box upload_file) }
```

Comment:
__[rubocop]__
* Line is too long. [142/130] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->